makefiles/Makefile.c_app

```diff
@@ -115,7 +115,13 @@ ifdef THREADED
     CXXFLAGS += $(THREADING_FLAGS)
 endif
 
-MPI_INC = $(foreach dir,$(shell $(MPICC) --showme:incdirs),-I$(dir))
+ifneq ($(strip $(MPI_INSTALL_PATH)),)
+    I_MPI_INC = -I$(MPI_INC)
```

Comment:
I don't think MPI_INC and MPI_LIB is present all the time. Only use that if they exist